friendship-quiz.js

```diff
@@ -12,14 +12,23 @@ window.onload = init;
 
 var answerList = []; //initialize answerList to be empty
 
-/* ADD YOUR CODE BELOW */
-
 //adds user-submitted answer to answerList
 function submitAnswer() {
-
+  var answer = document.getElementById("answerInput").value;
+  document.getElementById("answerInput").value = '';
+  answerList.push(answer);
 }
 
 //checks if user-submitted guess is in answerList
 function checkGuess() {
+  var guess = document.getElementById("guessInput").value;
+  var x = answerList.length;
```

Comment:
I probably wouldn't bother declaring a separate variable for this, and would just use `answerList.length` directly in the `for` loop condition.